internal/utils/miscellaneous_test.go

```diff
@@ -1,49 +1,62 @@
+// SPDX-FileCopyrightText: 2025 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
 package utils
 
 import (
 	"testing"
 	"time"
 
-	"sigs.k8s.io/controller-runtime/pkg/client"
-
 	. "github.com/onsi/gomega"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 func TestMergeMaps(t *testing.T) {
```

Comment:
You might need to rebase, since https://github.com/gardener/etcd-druid/pull/996 added these tests.

Reply:
its already been rebased, do you see any issues?